plumbing/format/packfile/packfile_test.go

```diff
@@ -277,3 +277,29 @@ func getIndexFromIdxFile(r io.Reader) idxfile.Index {
 
 	return idxf
 }
+
+func (s *PackfileSuite) TestSize(c *C) {
+	f := fixtures.Basic().ByTag("ref-delta").One()
+
+	index := getIndexFromIdxFile(f.Idx())
+	fs := osfs.New("")
+	pf, err := fs.Open(f.Packfile().Name())
+	c.Assert(err, IsNil)
+
+	packfile := packfile.NewPackfile(index, fs, pf)
+	defer packfile.Close()
+
+	// Get the size of binary.jpg, which is not delta-encoded.
+	offset, err := packfile.FindOffset(plumbing.NewHash("d5c0f4ab811897cadf03aec358ae60d21f91c50d"))
```

Comment:
Is there an easy way to dynamically get this hash without a static string? I couldn't figure out how it was determined by just looking at https://github.com/src-d/go-git-fixtures/blob/49a975e8b44a25af2e567da5982bb709d574e61c/fixtures.go

Reply:
I determined it out-of-band by checking out the test repo and using `git ls-tree`.  As far as I know there isn't a good way to do it in the context of this test, but I have it up for review with go-git too (src-d/go-git#1060), so we'll see if they suggest anything on that PR.

In the meantime, I'll merge this as-is, and if I fix up the test in the main repo, I'll fix it here too.